wagtail/search/tests/elasticsearch_common_tests.py

```diff
@@ -167,5 +170,58 @@ def test_slice_to_next_page(self):
         index.add_items(models.Book, books)
         index.refresh()
 
-        results = self.backend.search(None, models.Book)[110:]
+        results = self.backend.search(MATCH_ALL, models.Book)[110:]
         self.assertEqual(len(results), 53)
+
+    # Broken
+    @unittest.expectedFailure
+    def test_filter_in_values_list_subquery(self):
+        super().test_filter_in_values_list_subquery()
+
+    # Broken
+    @unittest.expectedFailure
+    def test_order_by_non_filterable_field(self):
+        super().test_order_by_non_filterable_field()
+
+    # Broken
+    @unittest.expectedFailure
+    def test_delete(self):
+        super().test_delete()
+
+    #
+    # Basic query classes
+    #
+
+    # Not implemented yet
+    @unittest.expectedFailure
```

Comment:
I think this should be implemented for Elasticsearch as well before merging into master